src/script/components/MessagesList/Message/ContentMessage/Warnings/PartialFailureToSend/PartialFailureToSend.tsx

```diff
@@ -43,8 +46,27 @@ const config = Config.getConfig();
 
 type ParsedUsers = {namedUsers: User[]; unknownUsers: QualifiedId[]};
 
-function generateNamedUsers(users: User[], userClients: QualifiedUserClients): ParsedUsers {
-  return Object.entries(userClients).reduce<ParsedUsers>(
+function generateNamedUsers(
+  users: User[],
+  userClientsOrQualifiedIds: QualifiedUserClients | QualifiedId[],
+): ParsedUsers {
+  if (Array.isArray(userClientsOrQualifiedIds)) {
+    return userClientsOrQualifiedIds.reduce<ParsedUsers>(
+      (parsedUsers, currentQulifiedId) => {
+        const user = users.find(user =>
+          matchQualifiedIds(user.qualifiedId, {id: currentQulifiedId.id, domain: currentQulifiedId.domain}),
```

Comment:
Can be simplified to just `matchQualifiedIds(user.qualifiedId, currentQulifiedId)`

Reply:
nice point 👍 
https://github.com/wireapp/wire-webapp/pull/15273/commits/eb0be37fcbba7f44bfa7975899c4668aa5562461